walletnode/services/senseregister/task.go

```diff
@@ -107,17 +100,14 @@ func (task *Task) run(ctx context.Context) error {
 		return task.nodes.WaitConnClose(ctx, nodesDone)
 	})
 
-	// get block height + hash
-	if err := task.getBlock(ctx); err != nil {
```

Comment:
I don't think should remove this?
This is a step to prepare for registration, put it in connectToTopRankNodes() is not really meaningful

Reply:
This function is put into connectToTopRankNodes() to follow the currennt design.